ui/components/component-library/picker-network/picker-network.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import { AvatarNetwork } from '../avatar-network';
+import { Icon, ICON_NAMES } from '../icon';
+import { Text } from '../text';
+import Box from '../../ui/box';
+import {
+  ALIGN_ITEMS,
+  COLORS,
+  DISPLAY,
+  SIZES,
+  BORDER_RADIUS,
+  TEXT,
+} from '../../../helpers/constants/design-system';
+
+export const PickerNetwork = ({
+  className,
+  avatarNetworkProps,
+  fallbackIconProps,
```

Comment:
I think we could use a slightly more appropriate prop name for the down arrow icon? Maybe simple `iconProps`?